### PR TITLE
added contributors section in README.md #10

### LINKED ISSUE #10: Add contributors section in README.md
Add contributors section in README.md using [all-contributors](https://github.com/all-contributors/all-contributors)

### PR DESCRIPTION
**Pull request by keremh**
Fix #10

### CODE REVIEW

**Comment by frinyvonnick** [comment]
Nice job @keremh 👏, could you add yourself too ?

**Comment by keremh** [comment]
Thank you @frinyvonnick , of course i will add my information too. :relaxed:

**Comment by frinyvonnick** [comment]
In this pull request ?

**Comment by keremh** [comment]
Can i add in another pull request?

**Comment by frinyvonnick** [comment]
Sure ! I'll merge this one as soon as possible :+1:

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # react-simple-infinite-loading
+[![All Contributors](https://img.shields.io/badge/all_contributors-4-orange.svg?style=flat-square)](#contributors-)
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
```
Comment (frinyvonnick) [inline README.md]: There are 5 badges could you keep only the first one ?

Comment (keremh) [reply README.md]: ok, i did it. Can you review code again?

**Comment by frinyvonnick** [comment]
@all-contributors please add @keremh for code

**Comment by frinyvonnick** [comment]
I just added the Github App for all contributors 😉

**Pull request merged**